open-api/rest-catalog-open-api.py

```diff
@@ -896,19 +896,6 @@ class SetPartitionStatisticsUpdate(BaseUpdate):
     )
 
 
-class TableRequirement(BaseModel):
```

Comment:
This should not be removed?

Reply:
It wasn't removed just moved down in the file due to `make generate`

https://github.com/apache/iceberg/pull/11287/files#diff-ffc6f59bffb61af69bc4ef19c7b28c9fefe015b2716b35db1deb6a5a884ca4feR1196